code/lib/cli/src/initiate.ts

```diff
@@ -313,22 +314,30 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
   }
   done();
 
-  const storybookInstalled = isStorybookInstalled(packageJson, options.force);
+  const storybookInstatiated = isStorybookInstatiated(resolve(process.cwd(), '.storybook'));
 
-  if (storybookInstalled && projectType !== ProjectType.ANGULAR) {
+  if (storybookInstatiated && projectType !== ProjectType.ANGULAR) {
     logger.log();
-    paddedLog('There seems to be a Storybook already available in this project.');
-    paddedLog('Apply following command to force:\n');
-    codeLog(['sb init [options] -f']);
-
-    // Add a new line for the clear visibility.
+    const { force } = await prompts([
+      {
+        type: 'confirm',
+        name: 'force',
+        message:
+          'We found a .storybook config directory in your project. Therefore we assume that Storybook is already instantiated for your project. Do you still want to continue and force the initialization?',
+      },
+    ]);
     logger.log();
-    throw new HandledError(`Angular project already installed`);
+    if (!force) {
+      throw new HandledError(`Project already inited with Storybook`);
```

Comment:
I think we don't need to throw an error anymore, the user is aware of the scenario and if they say "no" then it should just exit the process successfully

Reply:
I agree